flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -1030,6 +1030,17 @@ void createTemporarySystemFunction(
      */
     boolean dropTemporaryTable(String path);
 
+    /**
+     * Drops a table registered in the given path.
```

Comment:
```suggestion
     * Drops a table registered in the given path. Since views are considered virtual tables, it can also drop a registered view.
```
Is that correct? If yes, this sentence might avoid confusion.

Reply:
Oh, so DROP TABLE in SQL does the same thing?

Reply:
yep, even more `DROP TABLE` via `DropTableOpeartion` leads execution to the same code
https://github.com/apache/flink/blob/84044f45830542d13d6c9baeb2bfe30de3eac4ac/flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/ddl/DropTableOperation.java#L71-L75